app/views/miq_ae_tools/_import_export.html.haml

```diff
@@ -22,7 +22,18 @@
           :javascript
             $(":file").filestyle({icon: false, placeholder: "No file chosen"});
         .col-md-6
-          = submit_tag(_("Upload"), :id => "upload-datastore-import", :class => "upload btn btn-default")
+          = submit_tag(_("Upload"), :id => "upload-datastore-import", :class => "upload btn btn-default", :disabled => true)
+          :javascript
+            $("#upload_file").on("change",
+              function(){
+                if ($(this).val()){
+                  $('#upload-datastore-import').prop('disabled', false);
+                }
+                else {
+                  $('#upload-datastore-import').prop('disabled', true);
+                }
+              }
+            );
```

Comment:
I think this change is fine, but we should probably move it so that we don't have a bunch of logic sitting in the view. I'd propose to move this whole block into `app/assets/javascripts/import.js`, and into the `ImportSetup` object, and then just call something like `ImportSetup.setUpUploadDatastoreImportButton()` or something with a better name than I can come up with right now 😛 .

This way, we can also add specs.